src/common/sign-in/sign-in.tsx

```diff
@@ -0,0 +1,141 @@
+import { useState } from 'react';
+import { Button } from 'common/button';
+import { TextField, TextFieldStatus } from 'common/text-field';
+import { ReactComponent as EyeSVG } from '../../assets/icons/eye.svg';
+import { ReactComponent as ClosedEyeSVG } from '../../assets/icons/closed-eye.svg';
+import { ReactComponent as xSVG } from '../../assets/icons/x.svg';
+import { mailFormat, passwordFormat } from '../../helpers/utils.js';
+import styles from './sign-in.module.scss';
+
+type EmailType = {
+  inputValueEmail: string;
+  helperTextEmail: string;
+  fieldStatusEmail: TextFieldStatus;
+};
+
+const initEmailState = {
+  inputValueEmail: '',
+  helperTextEmail: '',
+  fieldStatusEmail: TextFieldStatus.default,
+};
+
+type PassType = {
+  inputValuePass: string;
+  helperTextPass: string;
+  fieldStatusPass: TextFieldStatus;
+};
+
+const initPassState = {
+  inputValuePass: '',
+  helperTextPass: '',
+  fieldStatusPass: TextFieldStatus.default,
+};
+
+export const SignIn = () => {
+  const [emailState, setEmailState] = useState<EmailType>(initEmailState);
+  const [passState, setPassState] = useState<PassType>(initPassState);
+  const [passHidden, setPassHidden] = useState(true);
+
+  const checkEmailValidation = (e: React.FocusEvent<HTMLInputElement>) => {
+    if (!e.target.value.match(mailFormat) && e.target.value !== '') {
+      setEmailState({
+        ...emailState,
+        helperTextEmail: 'Debe ser un email válido',
+        fieldStatusEmail: TextFieldStatus.error,
+      });
+    } else {
+      setEmailState({
+        ...emailState,
+        helperTextEmail: '',
+        fieldStatusEmail: TextFieldStatus.default,
+      });
+    }
+  };
+
+  const checkPassValidation = (e: React.FocusEvent<HTMLInputElement>) => {
+    if (e.target.value.length < 8 && e.target.value !== '') {
+      setPassState({
+        ...passState,
+        helperTextPass: 'La contraseña debe ser más larga que 8 caracteres',
+        fieldStatusPass: TextFieldStatus.error,
+      });
+    } else if (!e.target.value.match(passwordFormat) && e.target.value !== '') {
+      setPassState({
+        ...passState,
+        helperTextPass: 'La contraseña debe tener al menos un caracter especial y un caracter alfanumerico',
+        fieldStatusPass: TextFieldStatus.error,
+      });
+    } else {
+      setPassState({
+        ...passState,
+        helperTextPass: '',
+        fieldStatusPass: TextFieldStatus.default,
+      });
+    }
+  };
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.formDivision}>
+        <div className={styles.formContainer}>
+          <h1 className={styles.header1}>Iniciar sesión</h1>
+          <section className={styles.textField}>
```

Comment:
This can't be directly the textField and pass the style as a prop in the TextField like in the login?